examples/using-sqlx/Dockerfile

```diff
@@ -17,8 +17,8 @@ ADD --chown=rust:rust . ./
 RUN cargo build --release
 
 # Now, we need to build our _real_ Docker container, copying in `using-sqlx`.
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
+FROM debian:stretch-slim
+RUN apt-get update && apt-get install -y ca-certificates
```

Comment:
All images should continue to use Alpine, now that we do `ldd` checks on the host.

Many thanks for figuring out `ldd`, and for preparing this PR!